patroni/log.py

```diff
@@ -11,6 +11,12 @@
 _LOGGER = logging.getLogger(__name__)
 
 
+def debug_exception(logger_obj, msg, *args, **kwargs):
+    kwargs.pop("exc_info", False)
+    logger_obj.error(msg, *args, exc_info=False, **kwargs)
+    logger_obj.debug(msg, *args, exc_info=True, **kwargs)
```

Comment:
Basically that means we will see tracebacks only if log level is set to debug, which is too verbose.

Reply:
Only if we set log.traceback_level == debug, if we don't change log.traceback_level nothing changes.